example/android/app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,7 @@
     <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://exp.host/@lukaszkosmaty/example"/>
     <meta-data
         android:name="com.google.android.geo.API_KEY"
-        android:value="AIzaSyDbgaRNTr3PhYdj_PL7jY_o9u3R08Gf8Ao"/>
+        android:value="EMPTY_KEY"/>
```

Comment:
Until we implement some plugin to configure api keys for Google Maps on iOS and Android it will be necessary to manually put GooglePlaces Api Key here